templates/success.tmpl

```diff
@@ -0,0 +1,76 @@
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="https://s3.csh.rit.edu/csh-material-bootstrap/4.0.0-beta.3/dist/csh-material-bootstrap.min.css" media="screen">
+    <style>
+        html {
+            position: relative;
+            min-height: 100%;
+        }
+
+        body {
+            margin-bottom: 60px;
+            /* Margin bottom by footer height */
+        }
+
+        .footer {
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+            height: 60px;
+            /* Set the fixed height of the footer here */
+            line-height: 60px;
+            /* Vertically center the text there */
+            background-color: #f5f5f5;
+        }
+    </style>
+</head>
+
+<body>
+    <form action="/admin" method="POST">
+
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <div class="container">
+                <a class="navbar-brand" href="/upload">Plug</a>
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="/upload">Upload</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/admin">Admin <span class="sr-only">(current)</span></a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="col-lg-7">
+                    <div class="card mb-3">
+                    </div>
+                </div>
+            </div>
+
+            <div class="row justify-content-center">
+                <div class="col-lg-7">
+                    <div class="card mb-3">
+                        <h3 class="card-header">Success!</h3>
+                        <img style="width: 100%; display: block;" src="`+S3PresignPlug(plug).String()+`" alt="Card image">
+
+                        <div class="card-footer text-muted">
+                            This is how your Plug will appear on CSH sites. (This does not count towards the views for your Plug.)<br><br>Your Plug must be approved before it will appear for viewing. Any member of the following groups (drink, eboard, rtp) can do so via the admin page.
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </form>
+
+    <footer class="footer">
+        <div class="container">
+            <span class="text-muted">CSH Plug on <a href="https://github.com/liam-middlebrook/csh-plug">GitHub</a></span>
+        </div>
+    </footer>
+</body>
+
+</html>
```

Comment:
Please ensure your files end with newline characters, here's a StackOverflow post that links to the POSIX standard and provides a pretty solid explanation behind why this is important: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Also yes please for all things good use new lines at the end of files.  If it stays as is, the next person to modify this file with a sane editor will have to commit the correction.